config_test.go

```diff
@@ -288,3 +288,95 @@ func Test_stringsToMap(t *testing.T) {
 		})
 	}
 }
+
+func Test_objectErrors(t *testing.T) {
+	t.Parallel()
+
+	var c struct{}
+	if err := From("a non existent file").To(&c).Err(); err == nil {
+		t.Error("expected an error but got none.")
+	}
+
+	if err := From("a non existent file").From("another none existent file").FromEnv().To(&c).Err(); err == nil {
+		t.Error("expected an error but got none.")
+	}
+}
+
+func Test_conversionErrors(t *testing.T) {
```

Comment:
missing the `t.Parallel()` call for the function as a whole here.

https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721